packages/IMAPClient-Core.package/ICContact.class/properties.json

```diff
@@ -0,0 +1,15 @@
+{
+	"category" : "IMAPClient-Core",
+	"classinstvars" : [
+		 ],
+	"classvars" : [
+		 ],
+	"commentStamp" : "",
+	"instvars" : [
+		"email",
+		"name" ],
+	"name" : "ICContact",
+	"pools" : [
+		 ],
+	"super" : "Object",
+	"type" : "normal" }
```

Comment:
Does ICContact already have a class comment? If not, please add.

Reply:
Ok